opengrok-indexer/src/test/java/org/opengrok/indexer/search/context/HistoryContextTest.java

```diff
@@ -190,4 +198,37 @@ void testWriteMatch() throws IOException {
                 "r1=/foo%20bar/haf%2Bhaf@1\" title=\"diff to previous version\">diff</a> <b>foo</b>",
                 sb.toString());
     }
+
+    /**
+     * Test that inactive history entries are skipped when generating history context.
+     */
+    @Test
+    void testGetHistoryContextVsInactiveHistoryEntries() {
+        Set<String> filePaths = Set.of(File.separator + Paths.get("teamware", "foo.c"));
+        History history = new History(List.of(
```

Comment:
It would be nice to test some corner cases, like all history entries are inactive or if the next entry is not present etc.,  however not this PR.